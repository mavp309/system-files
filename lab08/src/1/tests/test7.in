-1.7321 -0.5774
1.4142 1.4142
