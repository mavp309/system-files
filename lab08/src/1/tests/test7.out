Sum: -0.3179+0.8368j
Difference: -3.1463+-1.9916j
Product: -1.6330+-3.2661j
Division: -0.8165+0.4083j
